packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -14,8 +14,6 @@ const trace = makeTracer('Reserve', false);
 
 const makeLiquidityKeyword = keyword => `${keyword}Liquidity`;
 
-const RunKW = 'RUN';
-
```

Comment:
Porque? It's not something that needs to be refactored. It's less Jessie-safe and @dckc has back-tracked on using them for keywords.

Reply:
BTW, @turadg, using variable keywords for object literals should be supported by Jessie.  If it isn't, that's a bug we need to fix in the Jessie spec, not something you need to avoid here.

Reply:
> using variable keywords for object literals should be supported by Jessie

By "Jessie-safe" I meant passes `@jessie-check`, which causes this lint error :
<img width="812" alt="Screen Shot 2022-06-14 at 11 53 55 AM" src="https://user-images.githubusercontent.com/21505/173667133-7f4eda2a-8842-47ab-86d7-add0b5615486.png">
 
Are you saying that's a bug in the Jessie linter?

To be clear, while that's a discussion of the drawback of using a manifest constant for keyword my main reason for pushing back on doing the keywords is it's more work (delaying release) for at best negligible value.

Reply:
> Please bring back the manifest constant rather than using 'Fee'.

I take it back.  We use the replacements for RUN as strings and keywords directly throughout much of this code, and it would be inconsistent to only require the manifest constant in Reserve, and a hassle to switch all of InterProtocol to manifest constants.